conf/leaf1.conf

```diff
@@ -7,16 +7,12 @@ service integrated-vtysh-config
 password opi
 enable password opi
 !
-vrf blue
- vni 100
-exit-vrf
-!
 vrf green
- vni 101
+ vni 100
```

Comment:
We tried to have a consistency between L3VNI and the bridge name (or vlan in case of vlan aware bridge) and since blue vrf doesn't have L3VNI anymore, we moved the ids for the other 2 vrfs. Since route table id is internal and cannot guarantee consistency unless we have a specific logic to ensure that, we didn't pay much attention to it. Can we address this via an issue if we think this really adds value or more clarity?

Reply:
IMO, VRF table-ids are internal route table numbers for a VRF whereas VNIs are exposed to the user, I dont see a reason to have a relation.

Reply:
Yes I agree the routing tables IDs can be implementation specific and auto-generated. ok let's leave it as it is for now